### DISCUSSION TITLE
2.3.2 - Cortex Error - FileInfo_7_0 Error

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#1743] [Unsupported Versions] [answered]
When trying to use the Analyzer FileInfo, I get this error

Traceback (most recent call last): File "/Cortex-Analyzers/analyzers/FileInfo/fileinfo_analyzer.py", line 3, in <module> import magic File "/usr/local/lib/python3.6/dist-packages/magic.py", line 181, in <module> raise ImportError('failed to find libmagic. Check your installation')ImportError: failed to find libmagic. Check your installation

Looks like libmagic is not installed. 

Not sure if the error means inside the docker image for TheHive/Cortex or on my Manager.

If it is inside my docker image, how do I update the image to include this?

**Comment by weslambert** [comment, marked as answer]
The requirements.txt for the FileInfo analyzer includes python-magic, and a pip3 list shows python-magic as installed inside of my container, so I'll have to take a closer look at why this may be happening.  I believe python-magic is just an interface to libmagic, so we may need to make sure it gets installed when we build the image.  I'l need to perform some testing, then I will follow up.

**Reply by weslambert** [reply]
I've pushed a fix that will be available in the next release, here: https://github.com/Security-Onion-Solutions/securityonion-image/pull/44.

**Comment by joeb1kenobe** [comment]
Thanks Wes, you guys are awesome as always!
